### ISSUE TITLE
Determine homepage content and layout

### ISSUE THREAD

**Issue by shanitajohn** [curationexperts/laevigata#508]
**STORY**
As a depositor or student visiting to the Emory ETD site, I would like the first page I arrive at to provide me with general information about ETD, explain a bit about the submission process, offer resources, allow me to begin the submission process, or simply search the repository.

**First draft wireframe of the ETD homepage:**
https://emory.box.com/s/fq3u0xzw9soxxtp4lcl4kohubzn00yhc

It has a place for all of the things that are on the current homepage: https://etd.library.emory.edu/

**Current Homepage Content**
- [x] Login link
- [x] Search bar
- [x] Image 
- [x] A few Recently published ETDs
- [x] All of the side navigation links now appear in the footer
- [x] Introductory copy 
- [x] News & Announcements 
- [x] Submission forms
- [x] Submit Your ETD button (Related to ticket #32)

**Comment by shanitajohn** [comment]
@bnash108 @fpici Can you please provide feedback on or approval on the suggested layout?

**Comment by shanitajohn** [comment]
![emory-wireframe-homepage-v1.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/56a796b4884b1f13002fff7c/125516c66e82c728ace21e0d46ed9995319bcebe86fbba45b65cfd91644e287e2c14a22c75c8f579fd0e39111d0014f9121c4d4eeefc817ff5be3a77df5b5db982395874c63c40f8bf844bb757a83c990dd4e8e8d51199a26867f5819d4c3f14041d89ff5b30a49f4ec15af44ff300.png)

**Comment by bnash108** [comment]
@shanitajohn I collected feedback from Fran and Lisa. We all really like the home page wireframe! We decided to tweak the intro text (above the "Submit Your ETD" button) a little. New text below:


The Emory Electronic Theses and Dissertations (ETD) Repository holds theses and dissertations from the Laney Graduate School, the Rollins School of Public Health, and the Candler School of Theology, as well as undergraduate honors papers from Emory College of Arts and Sciences. Emory University theses and dissertations submitted before the launch of the ETD repository can be found by using Emory University’s discoverE catalog <insert link>.
  
The theses and dissertations of the Wallace H. Coulter Department of Biomedical Engineering, a joint degree program by Emory University and Georgia Tech, are available in SMARTech <insert link>, Georgia Tech’s repository, and Emory University’s discoverE catalog <insert link>.
